packages/theme-addresses/README.md

```diff
@@ -0,0 +1,92 @@
+# @shopify/theme-addresses
+
+## Getting Started
+
+Theme Scripts can be used in any theme project. To take advantage of semantic versioning and easy updates, we recommend using NPM or Yarn to include them in your project:
+
+```
+yarn add @shopify/theme-addresses
+```
+
+and then import the functions you wish to use through ES6 imports:
+
+```
+import {CountryProvinceSelector} from '@shopify/theme-addresses`;
+```
+
+If you prefer not to use a package manager, you can download the latest version of Theme Product and include it in your project manually from the following links:
+
+- [theme-addresses.js](http://unpkg.com/@shopify/theme-addresses@latest/dist/theme-addresses.js)
+- [theme-addresses.min.js](http://unpkg.com/@shopify/theme-addresses@latest/dist/theme-addresses.min.js)
+
+## Usage
+
+theme-addresses.js helps developers work with common address releated operations, such as, building the country and province selectors
+
+### CountryProvinceSelector(countryOptions)
+
+Prepares an instance of a `CountryProvinceSelector` with the given country options
+
+- `countryOptions`: This is an html string generated by the following liquid tags:
+    * [`{{ all_country_option_tags | json }}`](https://help.shopify.com/en/themes/liquid/objects/all-country-option-tags)
+    * [`{{ country_option_tags | json }}`](https://help.shopify.com/en/themes/liquid/objects/country-option-tags)
+
+Example
+```js
+// In theme.liquid, somewhere before the end of </head> tag
+    ...
+    {% if template.directory == 'customers' %}
+      theme.allCountryOptionTags = {{ all_country_option_tags | json }};
+    {% endif %}
+  </script>
+  ...
+</head>
+```
+```js
+// In addresses.js
+import {CountryProvinceSelector} from '@shopify/theme-addresses';
+
+const billingCountryProvinceSelector = new CountryProvinceSelector(window.theme.allCountryOptionTags);
+```
+
+---
+
+### build(countryNodeElement, provinceNodeElement, options)
+
+Populates the given Select node element with country and province options.
+
+- `countryNodeElement`: The Select node element for countries
```

Comment:
The `<select>` element for countries

Is that easier to understand?

Reply:
yup!